includes/I18n.php

```diff
@@ -20,17 +20,17 @@ public function __construct() {
 	 * @return boolean
 	 */
 	public static function load_php_textdomain() {
-		return load_plugin_textdomain(
-			'wp-module-onboarding-data',
-			false,
-			dirname( container()->plugin()->basename ) . '/vendor/newfold-labs/wp-module-onboarding-data/languages'
-		);
+		if ( is_admin() || ( is_user_logged_in() && current_user_can( 'manage_options' ) ) ) {
```

Comment:
I see that earlier 'current_user_can' was checked if it was callable and not anymore, is it required?

Reply:
We will still need that check to ensure we are running this only for admin users. The `is_callable` check was removed because we are now running this code after the `init` hook.